src/app/containers/FrontPageSection/utilities/gridColumns.jsx

```diff
@@ -0,0 +1,10 @@
+const fullWidthColumns = {
```

Comment:
We can move this to the file where it's being used instead of having a single definition here.

Reply:
Done.